fixtures/build.js

```diff
@@ -22,28 +20,32 @@ window.EXAMPLE_BUILD = {
       }
     },
     "strategy": {
-      "type": "STI",
-      "stiStrategy": {
-        "builderImage": "openshift/ruby-20-centos7:latest",
-        "image": "openshift/ruby-20-centos7:latest"
+      "type": "Source",
+      "sourceStrategy": {
+        "from": {
+          "kind": "DockerImage",
```

Comment:
@spadgett, I think for the describer bits, always show the kind and name, and if the namespace exists, show it as well (it won't for DockerImage, and may not for the ImageStream* ones... I think current namespace is implied in that case). It's not that important for us to understand the format here (we're just displaying the data), but I was asking more for the overview case where we're correlating builds with deployment configs